packages/default-theme/cms/blocks/CmsBlockCategoryNavigation.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <CmsGenericElement
+    v-if="getContent"
+    :content="getContent"
+    class="cms-block-category-navigation"
+  />
+</template>
+
+<script>
+import CmsGenericElement from 'sw-cms/CmsGenericElement'
+
+export default {
+  components: {
+    CmsGenericElement,
+  },
+  name: 'CmsBlockCategoryNavigation',
+  props: {
+    content: {
+      type: Object,
+      default: () => ({}),
+    },
+  },
+  computed: {
+    getSlots() {
+      return this.content.slots || []
+    },
+    getContent() {
+      return this.getSlots.length && this.getSlots[0]
```

Comment:
why only the first one is picked up? is there any other slot available to get?

Reply:
Every block is treated individually. This particular block has only one slot in its definition so there is no possibility for him to have more slots :)

definition:
https://github.com/shopware/platform/blob/master/src/Administration/Resources/app/administration/src/module/sw-cms/blocks/sidebar/category-navigation/index.js